Classes/Session/Typo3Session.php

```diff
@@ -134,4 +136,15 @@ public function start(): Typo3Session {
   protected function getLifetime(): int {
     return 3600;
   }
+
+  protected function getSessionId(): string {
+    if (array_key_exists('formhandler_session', $_COOKIE)) {
```

Comment:
Add empty check here, just to be sure

`if (array_key_exists('formhandler_session', $_COOKIE) && !empty($_COOKIE['formhandler_session'])) {`